diff -ur rpm-analyzer-1.0-o/Makefile rpm-analyzer-1.0/Makefile
--- rpm-analyzer-1.0-o/Makefile	2006-02-21 11:09:24.000000000 -0700
+++ rpm-analyzer-1.0/Makefile	2006-02-21 22:55:38.000000000 -0700
@@ -25,13 +25,12 @@
 		exit 1; \
 	fi
 
-	mkdir -p $(DESTDIR)/usr/local/bin
+	mkdir -p $(DESTDIR)/usr/bin
 	mkdir -p $(DESTDIR)/$(RADATADIR)
 	mkdir -p $(DESTDIR)/$(RADATADIR)/package_mgr
 	install -m 644 src/*.py $(DESTDIR)/$(RADATADIR)
-	install -m 755 src/rpm-analyzer.py $(DESTDIR)/$(RADATADIR)
+	install -m 755 src/rpm-analyzer.py $(DESTDIR)/usr/bin/rpm-analyzer
 	install -m 644 src/package_mgr/*.py $(DESTDIR)/$(RADATADIR)/package_mgr
-	(cd $(DESTDIR)/usr/local/bin; ln -sf ../../../$(RADATADIR)/rpm-analyzer.py rpm-analyzer)
 
 install-man:
 	@if [ "$(MANDIR)" = "" ]; then \
